CHANGELOG.md

```diff
@@ -24,8 +24,9 @@ requesting a new account and will be displayed in the User Accounts module (PR #
 - Candidate's age can be retrieved from the Candidate class in days, months, or years (PR #5945)
 - Addition of autoSelect prop to React SelectElement allows for auto-selection of only available select option (PR #6156)
 - An `AcquisitionDate` field has been added to the `files` table (PR #6892)
+- The default value of the ScannerID field of the mri_protocol table is now NULL instead of 0 (PR #7496).
 #### Bug Fixes
-- *Add item here*
+- The default value of the ScannerID field of the mri_protocol table is now NULL instead of 0. This means that if a protocl is valid on all the study's scanners, then ScannerID=NULL (PR #XXX)
```

Comment:
```suggestion
- The default value of the `ScannerID` field of the `mri_protocol` table is now `NULL` instead of `0`. This means that if a protocol is valid on all the study's scanners, then `ScannerID` of the protocol should be set to `NULL` (PR #7496)
```